test/vif.jl

```diff
@@ -44,6 +44,10 @@ StatsModels.formula(model::Duncan) = model.formula
     @test vif(MyRegressionModel3()) ≈ [1, 1]
 end
 
+# Reference values from
+# https://github.com/palday/MixedModelsExtras.jl/blob/6db061cb856301c04b978101d4c53b90c890715b/test/vif.jl
```

Comment:
Can you copy the relevant parts from that file instead? It's weird to rely on MixedModelsExtras for that, especially if this PR is supposed to provide the same features more upstream.

Reply:
I mean, the relevant parts of that file are just the reference values. I'm planning on writing out the R code to do it again, which is why I hadn't merged this PR yet.

Reply:
added in https://github.com/JuliaStats/StatsModels.jl/pull/300/commits/96686ab44cdb921e14a120be8ea90ecb544237c4